src/components/Books.jsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import BookElement from './SingleBook';
 import AddBook from './AddBook';
 
```

Comment:
Buttons for adding and removing possible, as well as checking status are missing! 
- [ ] Kindly ensure that books can be added and removed, and buttons are visible.
- [ ] Also, ensure that the button for checking the status on the categories is present and the page works as expected.

**Your Home page:**
![image](https://user-images.githubusercontent.com/12745474/168145976-9629bed4-d6a1-478c-8949-a15cb678062f.png)

**Your categories page:**
![image](https://user-images.githubusercontent.com/12745474/168147759-1d53fc16-73cd-4c81-a149-9aeb78111136.png)